src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -834,16 +847,7 @@ internal static partial class Library
                     targetFunction: Substitute)
             },
             { BuiltinFunctionsCore.Switch, Switch },
-            {
-                BuiltinFunctionsCore.Table,
-                StandardErrorHandling<FormulaValue>(
-                    expandArguments: NoArgExpansion,
-                    replaceBlankValues: DoNotReplaceBlank,
-                    checkRuntimeTypes: DeferRuntimeTypeChecking,
-                    checkRuntimeValues: DeferRuntimeValueChecking,
-                    returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
-                    targetFunction: Table)
-            },
+            { BuiltinFunctionsCore.Table, Table },
             {
```

Comment:
The Table function can receive error parameters (and existing tests were expecting this). This change fixes it.